### PR TITLE
update: security info for personal token auth policy

### PR DESCRIPTION
**Pull request by staceysalamon-aiven**
## Checklist

- [X] The first paragraph of the page is on one line.
- [X] The other lines have a line break at 90 characters.
- [x] I checked the output.
- [ ] I applied the [style guide](../styleguide.md).
- [ ] My links start with `/docs/`.

### CODE REVIEW

File: docs/platform/howto/set-authentication-policies.md [hunk 1]
```diff
@@ -47,30 +47,38 @@ strongly recommended to enable this if you only have one Aiven organization.
...
 This token is tied to the password authentication method they logged in with.
-If the authentication policy changes later to only allow logging on with an identity
+If the authentication policy changes later to only allow logging in with an identity
 provider, then the token generated when the user was logged in with their password will
-not work. After logging in with an allowed method on the new authentication policy
+not work. After logging in with the allowed method on the new authentication policy
 the user can create a token.
 
+The authentication policy setting for personal tokens only applies to managed users.
+If **Allow managed users to create personal tokens** is disabled, managed users
```
Comment (italomg) [inline docs/platform/howto/set-authentication-policies.md:67]: We should probably change the text in the console before updating the docs, I can create a ticket for that in case it doesn't exist.

File: docs/platform/howto/set-authentication-policies.md [hunk 2]
```diff
@@ -47,30 +47,38 @@ strongly recommended to enable this if you only have one Aiven organization.
...
+cannot create personal tokens. Non-managed users can still create personal tokens,
+but they can't use them to access the organization's resources.
+
+Even with this setting disabled, managed users can still create personal tokens
```
Comment (italomg) [inline docs/platform/howto/set-authentication-policies.md:71]: when the setting is disabled managed users shouldn't be able to create personal tokens. However, if they have personal tokens before becoming managed users then they can still use those personal tokens for doing operations in other organizations besides the one that manages them(assuming those orgs allow personal tokens).

File: docs/platform/howto/set-authentication-policies.md [hunk 3]
```diff
@@ -47,30 +47,38 @@ strongly recommended to enable this if you only have one Aiven organization.
...
+in any other organization they are a member of that has it enabled. They can
+use the personal tokens from other organizations to access your organization's
+resources. To restrict the use of personal tokens from other organizations, select
```
Comment (italomg) [inline docs/platform/howto/set-authentication-policies.md:74]: > They can use the personal tokens from other organizations to access your organization's resources.

this sounds quite bad :disappointed: . The personal tokens are not associated with any organization per se, so maybe better say "They can use their personal tokens to access your organization's resources." or something along those lines.

File: docs/platform/howto/set-authentication-policies.md [hunk 4]
```diff
@@ -47,30 +47,38 @@ strongly recommended to enable this if you only have one Aiven organization.
...
 the user can create a token.
 
+The authentication policy setting for personal tokens only applies to managed users.
+If **Allow managed users to create personal tokens** is disabled, managed users
+cannot create personal tokens. Non-managed users can still create personal tokens,
+but they can't use them to access the organization's resources.
+
+Even with this setting disabled, managed users can still create personal tokens
+in any other organization they are a member of that has it enabled. They can
+use the personal tokens from other organizations to access your organization's
```
Comment (italomg) [inline docs/platform/howto/set-authentication-policies.md:73]: > They can use the personal tokens from other organizations to access your organization's resources.

this sounds quite bad :disappointed: . The personal tokens are not associated with any organization per se, so maybe better say "They can use their personal tokens to access your organization's resources." or something along those lines.

File: docs/platform/howto/set-authentication-policies.md (hunk 3 above)
Comment (italomg) [inline docs/platform/howto/set-authentication-policies.md:74]: > To restrict the use of personal tokens from other organizations, select **Require users to be logged in with an allowed authentication method**.

Considering my previous comment we might want to changes this also, maybe "To enforce that personal tokens used conform with your organization authentication policy select
**Require users to be logged in with an allowed authentication method**."

but then again, we should probably reword the option in the console :thinking:

File: docs/platform/howto/set-authentication-policies.md (hunk 1 above)
Comment (staceysalamon-aiven) [reply docs/platform/howto/set-authentication-policies.md:67]: I had thought we were going to keep Console copy the same, but agree that we should do that first and make it clearer! 💯

File: docs/platform/howto/set-authentication-policies.md (hunk 2 above)
Comment (staceysalamon-aiven) [reply docs/platform/howto/set-authentication-policies.md:71]: > if they have personal tokens before becoming managed users
Is this referring to personal tokens that were created in any organization?

File: docs/platform/howto/set-authentication-policies.md (hunk 4 above)
Comment (staceysalamon-aiven) [reply docs/platform/howto/set-authentication-policies.md:73]: It does, yeah, but I wanted to be clear that the _ability_ to create tokens is linked to the org auth policy. I'll figure out a way to reword this to make it clear that it's still a _personal_ token and not actually associated with the org, though.
